### ISSUE TITLE
Content is King

### ISSUE THREAD

**Issue by chingu1991** [chingu1991/TeamAwesomeDesign#10]
Make sure to create content for what we are passionated about for a portal portal/blog. In design, development, and etc. Post all links to content below/drafts for editing.

**Comment by andrewbiang888** [comment]
We are passionate about art. We are passionate about trying/learning new web concepts.

**Comment by andrewbiang888** [comment]
Our project management philosophy.
Our design philosophy.
Our development philosophy

**Comment by andrewbiang888** [comment]
Our background (passions, beliefs, failures, learned lessons)

**Comment by chingu1991** [comment]
@andrewbiang888 

Yes. Visual content as well. I think we should focus on making a shit ton of content.
